src/core/data.jl

```diff
@@ -1151,14 +1151,14 @@ function _correct_branch_directions!(pm_data::Dict{String,<:Any})
             branch_orginal = copy(branch)
             branch["f_bus"] = branch_orginal["t_bus"]
             branch["t_bus"] = branch_orginal["f_bus"]
-            branch["g_to"] = branch_orginal["g_fr"] .* branch_orginal["tap"]'.^2
-            branch["b_to"] = branch_orginal["b_fr"] .* branch_orginal["tap"]'.^2
-            branch["g_fr"] = branch_orginal["g_to"] ./ branch_orginal["tap"]'.^2
-            branch["b_fr"] = branch_orginal["b_to"] ./ branch_orginal["tap"]'.^2
-            branch["tap"] = 1 ./ branch_orginal["tap"]
-            branch["br_r"] = branch_orginal["br_r"] .* branch_orginal["tap"]'.^2
-            branch["br_x"] = branch_orginal["br_x"] .* branch_orginal["tap"]'.^2
-            branch["shift"] = -branch_orginal["shift"]
+            branch["g_to"] = branch_orginal["g_fr"] #.* branch_orginal["tap"]'.^2
+            branch["b_to"] = branch_orginal["b_fr"] #.* branch_orginal["tap"]'.^2
+            branch["g_fr"] = branch_orginal["g_to"] #./ branch_orginal["tap"]'.^2
+            branch["b_fr"] = branch_orginal["b_to"] #./ branch_orginal["tap"]'.^2
+            # branch["tap"] = 1 ./ branch_orginal["tap"]
+            branch["br_r"] = branch_orginal["br_r"] #.* branch_orginal["tap"]'.^2
+            branch["br_x"] = branch_orginal["br_x"] #.* branch_orginal["tap"]'.^2
+            # branch["shift"] = -branch_orginal["shift"]
```

Comment:
We can go ahead and remove the references to "tap", instead of just commenting them out, they are just dangling references now.

Reply:
Okay, I wanted to check this first with you